SnoopCompileBot/Project.toml

```diff
@@ -0,0 +1,31 @@
+name = "SnoopCompileBot"
+uuid = "1d5e0e55-7d74-4714-b8d8-efa80e938cf7"
+author = ["Tim Holy <tim.holy@gmail.com>"]
```

Comment:
We should change this to you.

Reply:
Thank you! https://github.com/timholy/SnoopCompile.jl/pull/98/commits/99001babd8e4e78696e22fb47c58a16f77b4299d